qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -190,15 +198,114 @@ public void test022InstallPluginsFromLocalArchive() {
         // Stuff the proxy settings with garbage, so any attempt to go out to the internet would fail
         sh.getEnv()
             .put("ES_JAVA_OPTS", "-Dhttp.proxyHost=example.org -Dhttp.proxyPort=9999 -Dhttps.proxyHost=example.org -Dhttps.proxyPort=9999");
-        sh.run("/opt/plugins/plugin-wrapper.sh install --batch analysis-icu");
+        sh.run(bin.pluginTool + " install --batch analysis-icu");
 
         plugins = sh.run(bin.pluginTool + " list").stdout.lines().collect(Collectors.toList());
 
         assertThat("Expected " + plugin + " to be installed", plugins, hasItems(plugin));
     }
 
     /**
-     * Check that the JDK's cacerts file is a symlink to the copy provided by the operating system.
+     * Checks that ESS images can manage plugins by deploying a plugins config file.
+     */
+    public void test023InstallPluginsUsingConfigFile() {
+        assumeTrue("Only applies to ESS images", distribution().packaging == Packaging.DOCKER_CLOUD_ESS);
```

Comment:
It's only the test that is being restricted, because only the Cloud ESS image ships with extra plugins that we can easily install.

Reply:
Right, I mean the declarative plugin installation stuff in general. Do we not want to allow _all_ Docker users to be able to install plugins via the config file? If so, shouldn't we test/document that across all Docker image variants?

Reply:
Actually yes - for example, ECK uses the default images, and we want ECK to be able to use declarative plugins, so we need to test across image variants. I'll try to figure something out so that we can test plugin installation without relying on the image already containing plugins to install.

Reply:
Absolutely. Currently ECK suggests [using an init container](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-bundles-plugins.html) to install plugins. Given our conversation in our sync today, this seems like another thing we should have cloud eyes on before we pull the trigger.